src/binary.ts

```diff
@@ -216,17 +216,18 @@ export class Binary {
   }
 
   toJSON(): string {
-    return this.buffer.toString('base64');
+    return ByteUtils.toBase64(this.buffer);
   }
 
-  toString(format?: string): string {
-    return this.buffer.toString(format);
+  // TODO: BREAKING
+  toString(): string {
+    return ByteUtils.toText(this.buffer);
   }
```

Comment:
This wasn't predicted in our set of breaking changes but I think its an additional one we'll want to make, we shouldn't implement every format that Node supports. The workaround is to call to string on the buffer directly.